examples/example_plugin/example_plugin/static/example_plugin/docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mkdocs==1.2.3
```

Comment:
Why do we have both `examples/example_plugin/docs/requirements.txt` and `examples/example_plugin/example_plugin/static/example_plugin/docs/requirements.txt`?

Reply:
I am guessing that was done by mkdocs... I did not manually add anything to static/example_plugin/docs.